### PR TITLE
Admin product detail delete

### PR DESCRIPTION
**Pull request by namuetn**
## Related Tickets:
- [#28912](https://edu-redmine.sun-asterisk.vn/issues/28912)

## WHAT this PR does?
- delete product detail

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by namuetn** [comment]
ready

File: resources/views/fashi/admin/product/create.blade.php [hunk 1]
```diff
@@ -23,10 +23,10 @@
 
                 <div class="form-group form-product">
                     <label for="category" class="col-form-label">{{ trans('text.category') }}:</label>
-                    <select class="form-control" name="category" id="category" value="{{ old('parent_id') }}">
+                    <select class="form-control" name="category" id="category" value="{{ old('category') }}">
                         <option value="">{{ trans('text.category') }}</option>
                         @foreach ($categories as $category)
-                            @if($category->parent_id != null)
+                            @if(($category->parent_id != null) || ($category->id == old('category')))
```
Comment (hoanglm-2435) [inline resources/views/fashi/admin/product/create.blade.php]:
```suggestion
                            @if ($category->parent_id != null || $category->id == old('category'))
```

File: resources/views/fashi/admin/product/create.blade.php [hunk 2]
```diff
@@ -23,10 +23,10 @@
...
                                 <option
                                   value="{{ $category->id }}">
```
Comment (hoanglm-2435) [inline resources/views/fashi/admin/product/create.blade.php]: indent 4 spaces

File: resources/views/fashi/admin/product/edit.blade.php [hunk 3]
```diff
@@ -62,15 +62,32 @@
...
+                <div class="form-group form-product wrapper">
+                    <div>
+                        <label for="color" class="col-form-label">{{ trans('text.color') }}:</label>
+                        <select class="form-control dis col-md-2 d-sm-inline mt-3 mr-4" name="colors[]" id="color" value="{{ old('parent_id') }}">
+                            <option value="red">{{ trans('text.red') }}</option>
+                            <option value="black">{{ trans('text.black') }}</option>
+                            <option value="yellow">{{ trans('text.yellow') }}</option>
+                            <option value="green">{{ trans('text.green') }}</option>
+                            <option value="blue">{{ trans('text.blue') }}</option>
+                            <option value="violet">{{ trans('text.violet') }}</option>
```
Comment (hoanglm-2435) [inline resources/views/fashi/admin/product/edit.blade.php]: đưa value vào config

File: resources/views/fashi/admin/product/edit.blade.php [hunk 4]
```diff
@@ -62,15 +62,32 @@
...
+                        </select>
+
+                        <label for="size" class="col-form-label">{{ trans('text.size') }}:</label>
+                        <select class="form-control col-md-2 d-sm-inline mt-3 mr-4" name="sizes[]" id="size" value="{{ old('parent_id') }}">
+                            <option value="s">S</option>
+                            <option value="m">M</option>
+                            <option value="l">L</option>
+                            <option value="xs">XS</option>
```
Comment (hoanglm-2435) [inline resources/views/fashi/admin/product/edit.blade.php]: tương tự

File: resources/views/fashi/admin/product/create.blade.php [hunk 5]
```diff
@@ -13,7 +13,7 @@
             <div class="card-body">
                 <div class="form-group form-product">
                     <label for="name" class="col-form-label">{{ trans('text.name') }}:</label>
-                    <input type="text" class="form-control  @error('name') is-invalid @enderror" id="name" name="name" value="{{ old('name') }}">
+                    <input type="text" class="form-control  @error('name') is-invalid @enderror" id="name" name="name" value="{{ old('name') }}" required="">
```
Comment (thaonp104) [inline resources/views/fashi/admin/product/create.blade.php]:
```suggestion
                    <input type="text" class="form-control  @error ('name') is-invalid @enderror" id="name" name="name" value="{{ old('name') }}" required="">
```
bên dưới co lỗi này thì sửa tương tự nhé

**Comment by namuetn** [comment]
ready

**Pull request merged**